packages/components/src/globals/scss/__tests__/css--font-face-test.js

```diff
@@ -9,56 +9,66 @@
 
 const { createSassRenderer } = require('@carbon/test-utils/scss');
 
-const render = createSassRenderer(__dirname);
+const tests = useDartSass => {
+  const render = createSassRenderer(__dirname, '', useDartSass).renderer;
 
-describe('_css--font-face.scss', () => {
-  it('should not output CSS if $css--font-face is false', async () => {
-    const { result } = await render(`
+  let compiler = ' in node-sass';
+  if (useDartSass) {
+    compiler = ' in dart-sass';
+  }
+
+  describe(`_css--font-face.scss${compiler}`, () => {
+    it('should not output CSS if $css--font-face is false', async () => {
+      const { result } = await render(`
 $css--reset: false;
 $css--font-face: false;
 @import '../css--font-face';
 `);
 
-    // Should be an empty string, currently will output only @keyframes that are
-    // not wrapped around a css flag
-    expect(result.css.toString()).toMatchSnapshot();
-  });
+      // Should be an empty string, currently will output only @keyframes that are
+      // not wrapped around a css flag
+      expect(result.css.toString()).toMatchSnapshot();
+    });
 
-  it('should output helvetica if $css--font-face is true and $css--plex is false', async () => {
-    const { result } = await render(`
+    it('should output helvetica if $css--font-face is true and $css--plex is false', async () => {
+      const { result } = await render(`
 $css--reset: false;
 $css--font-face: true;
 $css--plex: false;
 @import '../css--font-face';
 `);
 
-    expect(result.css.toString()).toEqual(
-      expect.stringContaining('@font-face')
-    );
-    expect(result.css.toString()).toEqual(
-      expect.stringContaining(`font-family: 'IBM Helvetica'`)
-    );
-  });
+      expect(result.css.toString()).toEqual(
+        expect.stringContaining('@font-face')
+      );
+      expect(result.css.toString()).toEqual(
+        expect.stringMatching(/font-family: ['"]IBM Helvetica['"]/)
```

Comment:
dart-sass seems to prefer using double quotes `"`, so I just changed this to match either - generally it'll behave the same as `stringContaining`